charts/rode-ui/templates/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" ($.Chart.AppVersion) ) }}"
```

Comment:
I'm thinking the v is something we should have pulled out of the ui version honestly.

Reply:
I like to have it on the Git tags, maybe not on the Docker image tags.

Reply:
I still like the `v` prefix on image tags as long as you recognize that the `v` is not part of the version. hence the need for the string format